objects/obj_controller/Step_0.gml

```diff
@@ -194,421 +171,561 @@ if (x<800) then x=800;
-                if (ma_role[sel]=obj_ini.role[100,17]) or (ma_role[sel]=string(obj_ini.role[100,17])+" Aspirant") or (ma_role[sel]="Chief "+string(obj_ini.role[100,17])) then ui_specialist=7;// Librarian
-                if (ma_role[sel]="Death Company") then ui_specialist=15;// Death Company
-                
-                if (global.chapter_name="Dark Angels"){
-                    if (ma_role[sel]=obj_ini.role[100,4]) then ui_coloring="bone";
-                    if (managing=1){
-                        if ((string_count("Terminator",ma_armour[sel])>0) or (ma_armour[sel]="Tartaros")){
-                            if (ma_role[sel]=obj_ini.role[100,5]) then ui_coloring="bone";
-                            if (ma_role[sel]=obj_ini.role[100,7]) then ui_coloring="bone";
-                            if (ma_role[sel]="Standard Bearer") then ui_coloring="bone";
+                if (ma_gear[sel]=="Rosarius") then damage_res+=0.33;
+                if (ma_gear[sel]=="Iron Halo"){
+                    damage_res+=0.33;
+                    ach+=20;
+                }
+                if (ma_mobi[sel]=="Jump Pack"){
+                    damage_res+=0.1;
+                }
+                if (ma_mobi[sel]=="Bike") then ach+=25;
+                if (ma_wep1[sel]=="Boarding Shield"){
+                    ach+=20;
+                    acy+=4;
+                }
+                if (ma_wep2[sel]=="Boarding Shield"){
+                    ach+=20;
+                    acy+=4;
+                }
+                if (ma_wep1[sel]=="Storm Shield"){
+                    ach+=30;
+                    acy+=8;
+                }
+                if (ma_wep2[sel]=="Storm Shield"){
+                    ach+=30;
+                    acy+=8;
+                }
+                if (ma_armour[sel]=="MK3 Iron Armour") then ranged_attack-=0.1;
+                if (ma_armour[sel]=="MK4 Maximus"){
+                    ranged_attack+=0.05;
+                    melee_attack+=0.05;
+                }
+                // heresy should be lower damage resistance, lowered ap for now so it's easier for players to digest
+                if (ma_armour[sel]=="MK5 Heresy"){
+                    melee_attack+=0.2;
+                    ranged_attack-=0.05;
+                }
+                if (ma_armour[sel]=="MK6 Corvus"){
+                    ranged_attack+=0.1;
+                }
+                if (string_count("Artificer",ma_armour[sel])>0){
+                    melee_attack+=0.1;
+                }
+                if (string_count("Terminator",ma_armour[sel])>0){
+                    ranged_attack-=0.1;
+                    melee_attack+=0.2;
+                }
+                if (ma_armour[sel]=="Tartaros"){
+                    ranged_attack-=0.05;
+                    melee_attack+=0.2;
+                }
+
```

Comment:
add to trello that stat mechanics effects of equipment should be coded into some sort of equipment data file?